<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>461</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Estado dispositivos</string>
    </property>
    <widget class="QPushButton" name="enviarComando">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Enviar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="comando">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuPrincipal">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionGuardar"/>
   </widget>
   <widget class="QMenu" name="menuDispositivos">
    <property name="title">
     <string>Dispositivos</string>
    </property>
    <widget class="QMenu" name="menuArduino">
     <property name="title">
      <string>Arduino</string>
     </property>
     <widget class="QMenu" name="menuPuertos">
      <property name="title">
       <string>Puerto</string>
      </property>
      <addaction name="actionSeleccionarPuerto"/>
      <addaction name="separator"/>
     </widget>
     <addaction name="conectarArduino"/>
     <addaction name="actionDesconectar"/>
     <addaction name="menuPuertos"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuArduino"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuDispositivos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="conectarArduino">
   <property name="text">
    <string>Conectar</string>
   </property>
  </action>
  <action name="actionj_2">
   <property name="text">
    <string>j</string>
   </property>
  </action>
  <action name="actionCOM_1">
   <property name="text">
    <string>COM 1</string>
   </property>
  </action>
  <action name="actionDesconectar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Desconectar</string>
   </property>
  </action>
  <action name="actionSeleccionarPuerto">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Seleccionar Puerto</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
